src/Components/SmartComponents/Systems/Systems.js

```diff
@@ -98,7 +106,8 @@ class Systems extends React.Component {
 
     apply = (config = {}) => {
         this.setState({ ...this.state, ...config }, () => {
-            const params = omit(this.state, ['InventoryCmp', 'columns', 'removeListener']);
+            const params = omit(this.state,
```

Comment:
Let's try to get rid of lodash omit - performance issues. Let's use object spread destructuring to achieve the same result.

Reply:
Done also for other occurrences.